### PR TITLE
Update install-petra-wallet.md to link to Petra.app docs

### PR DESCRIPTION
**Pull request by clay-aptos**
### Description

Better intertwine these docs sets. Make some small edits along the way.

### Test Plan

Manual test of links

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4766)

### CODE REVIEW

**Review by rajkaramchedu — APPROVED** [review]

File: developer-docs-site/docs/guides/install-petra-wallet.md [hunk 1]
```diff
@@ -5,24 +5,38 @@ slug: "install-petra-wallet-extension"
...
 
 ## Install from the Chrome Web Store
 
+For the latest stable release, install the extension from the Chrome Web Store:
+
 1. Visit the [Petra Wallet extension page](https://chrome.google.com/webstore/detail/petra/ejjladinnckdgjemekebdpeokbikhfci).
-2. Click the **Add to Chrome** button.
+1. Click the **Add to Chrome** button.
+1. Find your Petra Wallet in your Chrome extensions anytime by loading this in the browser location (URL) bar: chrome://extensions/
+1. Optionally, [pin the extension](https://support.google.com/chrome/a/answer/11190170) to make it findable by Petra button in your Chrome browser toolbar.
```
Comment (hariria) [inline developer-docs-site/docs/guides/install-petra-wallet.md:17]: to make it findable by Petra button in your Chrome browser toolbar

to

to make it findable in your Chrome browser toolbar

File: developer-docs-site/docs/guides/install-petra-wallet.md [hunk 2]
```diff
@@ -5,24 +5,38 @@ slug: "install-petra-wallet-extension"
...
+For the latest development release, install the extension from GitHub:
+
+1. Download the latest [Petra Wallet release](https://github.com/aptos-labs/aptos-core/releases?q=wallet&expanded=true) and unzip it.
+1. Open a Chrome window and navigate to the *Extensions* using any of the below methods:
+    - At the top-right corner of the browser window, click the three vertical dots and then **More tools** > **Extensions**
+    - On a new tab or window type `chrome://extensions` in the URL field and press return.
+1. Enable **Developer mode** at the top right of the *Extensions* page.
+1. Click **Load unpacked** at the top left and point it to the `build` folder where you just unzipped the downloaded Wallet release.
+1. Find your Petra Wallet in your Chrome extensions anytime by loading this in the browser location (URL) bar: chrome://extensions/
+1. Optionally, [pin the extension](https://support.google.com/chrome/a/answer/11190170) to make it findable by Petra button in your Chrome browser toolbar.
```
Comment (hariria) [inline developer-docs-site/docs/guides/install-petra-wallet.md:30]: These are all 1.?

File: developer-docs-site/docs/guides/install-petra-wallet.md [hunk 3]
```diff
@@ -5,24 +5,38 @@ slug: "install-petra-wallet-extension"
...
 
-Now you will see Wallet in your Chrome extensions.
+See the [Petra docs](https://petra.app/docs/petra-intro) for developer and user instructions.
 
 :::tip
 If you would like to restore an existing Aptos wallet account in Petra, you must first create a new account in Petra, then you can switch and import the account.
 :::
+
+## Get help
```
Comment (hariria) [inline developer-docs-site/docs/guides/install-petra-wallet.md:38]: Lmfao maybe we can word this something other than "Get help" :D

File: developer-docs-site/docs/guides/install-petra-wallet.md [hunk 4]
```diff
@@ -5,24 +5,38 @@ slug: "install-petra-wallet-extension"
...
+
+Find additional documentation at: [petra.app](https://petra.app/docs/petra-intro)
```
Comment (hariria) [inline developer-docs-site/docs/guides/install-petra-wallet.md:40]: at: [petra.app]...

to

at [petra.app]

**Comment by rajkaramchedu** [comment]
> _this works flawlessly in GitHub Markdown. Any issues with Docusaurus and our site?_

Preview shows all okay, so we are good. In the past when I used this it generated such questions from the reviewers, and I didn't like how it added to the review traffic, so I stopped using it. I am okay either way, low care factor for me.

**Review by kent-white — APPROVED** [review]

**Pull request merged**